src/main/views/list.njk

```diff
@@ -0,0 +1,77 @@
+{% extends "template.njk" %}
```

Comment:
should we rename the file to something more understandable than `list`?

Reply:
btw i dont mean the line ive commented on, i mean the name of this actual file

Reply:
Updated to hearingList :)